Clear Project Guidelines Attempt 2

Computer8543 opened the pull request:
I addressed your comments in the Clear Project Guidelines thread in this one. If you have any more comments to make on this push request before merging, feel free to do so. I am learning a lot by doing this.

abkaufmann reviewed (commented):
I really appreciated this reflection of the exercises and found that some of the commentary matched my own thoughts - some of which were stated in my reflection, and some of which were still unvoiced.  It is interesting to see how the dialog has brought about alignment in these areas.

billwestfall reviewed (approved):
Michael gave a great reflection on his green belt request, including going over the topics of judgement, conflict, models, and more.
We first covered the topic of judgement vs analysis, and the value of going deeper on issues, vs superficial judgement.
Michael went over some of the models that are covered in the green belt (Cynefin and spiral dynamics), and how we can get the value out of investigating scenarios with these frameworks.
We also covered the new movie requirement and the value it brings as far as emotions when involved with situations.
Overall a great discussion and very insightful! Congratulations Michael!
+1

Computer8543 committed:
Update kyle-ingersoll.yaml

Changed Base 64 String to Central Repository String to resolve diff conflict in file.

Computer8543 committed:
Create LICENSE.md

Computer8543 committed:
Create README.md

Computer8543 committed:
Address DRY issue through directly linking to LICENSE.md

Computer8543 committed:
Made README.md intro more mysterious.
